src/NexyCrypt.php

```diff
@@ -79,13 +79,21 @@ public function __construct($privateKeyPath = null, $endpoint = null)
     }
 
     /**
-     * Generates or read privates key and starts registration.
+     * Generate private key.
      */
-    public function register()
+    public function createKey()
```

Comment:
Maybe just `create`? This looks more consistent with the other methods like `register`.